src/tree.ts

```diff
@@ -130,10 +130,13 @@ export class NodeType {
 
   /// Returns true when this node's name or one of its
   /// [groups](#tree.NodeProp^group) matches the given string.
-  is(name: string) {
-    if (this.name == name) return true
-    let group = this.prop(NodeProp.group)
-    return group ? group.indexOf(name) > -1 : false
+  is(name: string | number) {
+    if (this.name === name || this.id === name) return true
+    if(typeof name === 'string'){
```

Comment:
Since we're going to need to do this `typeof` check anyway, I'd prefer to also move the initial comparison into the conditional.

And please try to use a code style similar to the surrounding code—space after `if` and before the brace, and `==` instead of `===` whenever possible.

Reply:
thanks for the review :). Is it better like this ?

Reply:
just in case, we can replace the condition `group ? group.indexOf(name) > -1 : false` by just `group && group.indexOf(name) > -1`. It will avoid the ternary condition. 

I can change it as well if you want